templates/firefox-ios.xliff

```diff
@@ -2717,6 +2540,61 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Client/Intro.strings" source-language="en" datatype="plaintext">
```

Comment:
These have moved but the filename and the ids should be the same.

Reply:
Hmm Github is being super weird to me today.
So once more: yes, confirming this is good!